src/Neo.Cryptography.BLS12_381/Neo.Cryptography.BLS12_381.csproj

```diff
@@ -2,9 +2,14 @@
 
   <PropertyGroup>
     <VersionPrefix>0.2.0</VersionPrefix>
-    <TargetFramework>net7.0</TargetFramework>
+    <TargetFrameworks>netstandard2.1;net7.0</TargetFrameworks>

```

Comment:
The other changes are related to this one?

Reply:
yes, dotnet core and standard support different methods, have to update them.